ush/forecast_predet.sh

```diff
@@ -528,15 +528,32 @@ FV3_predet(){
 
   # Aerosol options
   IAER=${IAER:-1011}
-
+  MERRA2_6ym=${MERRA2_6ym:-".false."}
   ## merra2 aerosol climo
   if (( IAER == 1011 )); then
-    local month mm
-    for (( month = 1; month <=12; month++ )); do
-      mm=$(printf %02d "${month}")
-      cpreq "${FIXgfs}/aer/merra2.aerclim.2014-2023.m${mm}.nc" "aeroclim.m${mm}.nc"
-    done
-  fi
+    if [[ "${MERRA2_6ym}" == ".false." ]]; then
+#   local month mm
+      for (( month = 1; month <=12; month++ )); do
+        mm=$(printf %02d "${month}")
+        ${NCP} "${FIXgfs}/aer/20220805/merra2.aerclim.2014-2023.m${mm}.nc" "aeroclim.m${mm}.nc"
```

Comment:
should use "cpreq" instead of ${NCPP}. In addition, ~sorc/link_workflow.sh only links one aer fix version. It has to link two versions with this PR update